client/src/component/AvailabilityWidget.js

```diff
@@ -2,99 +2,122 @@ import React from "react";
-    },
-    forOuterBox: {
-      height: "100%",
-      display: "flex",
-      flexDirection: "column",
-      alignItems: "center",
-      justifyContent: "center",
-    },
-    formControl: {
-        margin: theme.spacing(1),
-        minWidth: 120,
-    },
-    selectEmpty: {
-        marginTop: theme.spacing(2),
-    },
-    url_input: {
-        '& > *': {
-          margin: theme.spacing(1),
-          width: '25ch',
-        },
-      },
-    url_prefix: {
-        color: "#a8b1c7",
-    },
-    divider: {
-        height: 28,
-        margin: 4,
+  forPaper: {
+    margin: theme.spacing(2),
+    width: theme.spacing(75),
+    height: theme.spacing(55),
+  },
+  forOuterBox: {
+    height: "100%",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  formControl: {
+    margin: theme.spacing(1),
+    minWidth: 120,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing(2),
+  },
+  url_input: {
+    "& > *": {
+      margin: theme.spacing(1),
+      width: "25ch",
     },
+  },
+  url_prefix: {
+    color: "#a8b1c7",
+  },
+  divider: {
+    height: 28,
+    margin: 4,
+  },
 }));
 
 const timeOptions = [
-    '00:00', '00:30',
-    '01:00', '01:30',
-    '02:00', '02:30',
-    '03:00', '03:30',
-    '04:00', '04:30',
-    '05:00', '05:30',
-    '06:00', '06:30',
-    '07:00', '07:30',
-    '08:00', '08:30',
-    '09:00', '09:30',
-    '10:00', '10:30',
-    '11:00', '11:30',
-    '12:00', '12:30',
-    '13:00', '13:30',
-    '14:00', '14:30',
-    '15:00', '15:30',
-    '16:00', '16:30',
-    '17:00', '17:30',
-    '18:00', '18:30',
-    '19:00', '19:30',
-    '20:00', '20:30',
-    '21:00', '21:30',
-    '22:00', '22:30',
-    '23:00', '23:30',
+  "00:00",
+  "00:30",
+  "01:00",
+  "01:30",
+  "02:00",
+  "02:30",
+  "03:00",
+  "03:30",
+  "04:00",
+  "04:30",
+  "05:00",
+  "05:30",
+  "06:00",
+  "06:30",
+  "07:00",
+  "07:30",
+  "08:00",
+  "08:30",
+  "09:00",
+  "09:30",
+  "10:00",
+  "10:30",
+  "11:00",
+  "11:30",
+  "12:00",
+  "12:30",
+  "13:00",
+  "13:30",
+  "14:00",
+  "14:30",
+  "15:00",
+  "15:30",
+  "16:00",
+  "16:30",
+  "17:00",
+  "17:30",
+  "18:00",
+  "18:30",
+  "19:00",
+  "19:30",
+  "20:00",
+  "20:30",
+  "21:00",
+  "21:30",
+  "22:00",
+  "22:30",
+  "23:00",
+  "23:30",
 ];
 
 const AvailabilityWidget = (props) => {
-    const classes = useStyles();
-    const [startTime, setStartTime] = React.useState(["09:00"]);
-    const [endTime, setEndTime] = React.useState(["17:00"]);
-    const [daysSelected, setDays] = React.useState({
-        'Sundays':false,  
-        'Mondays':true, 
-        'Tuesdays':true, 
-        'Wednesdays':true, 
-        'Thursdays':true, 
-        'Fridays':true, 
-        'Saturdays':false
-    });
+  const classes = useStyles();
+  const [startTime, setStartTime] = React.useState(["09:00"]);
+  const [endTime, setEndTime] = React.useState(["17:00"]);
+  const [daysSelected, setDays] = React.useState({

```

Comment:
Can you import useState at the beginning of the file instead of using React.useState?